### PR TITLE
[NP-7869] Various wizard-related fixes

### PR DESCRIPTION
**Pull request by KernelDeimos**

### CODE REVIEW

**Review by jlhughes — COMMENTED** [review]
LGTM

File: src/foam/u2/wizard/wizardflow/AddWizardlet.js [hunk 1]
```diff
@@ -61,12 +61,13 @@ foam.CLASS({
...
-              isAvailable: true,
-              customView: {
-                class: 'foam.u2.Element'
-              }
-            }, x)
+          title: newWizardlet.title,
+          isAvailable: true,
+          customView: {
+            class: 'foam.u2.detail.FlexSectionedDetailView'
+            // class: 'foam.u2.LoadingSpinner',
```
Comment (sarthak-marwaha) [inline src/foam/u2/wizard/wizardflow/AddWizardlet.js]: remove comment

**Review by sarthak-marwaha — COMMENTED** [review]
One minor comment otherwise LGTM

**Comment by kgrgreer** [comment]
LGTM

File: src/foam/u2/wizard/wizardflow/AddWizardlet.js (hunk 1 above)
Comment (KernelDeimos) [inline src/foam/u2/wizard/wizardflow/AddWizardlet.js]:
```suggestion
```

**Review by Nauna — APPROVED** [review]
everyone already approved - so just following ;)

**Pull request merged**